client/Windows/wf_client.c

```diff
@@ -783,7 +783,7 @@ static int freerdp_client_focus_out(wfContext* wfc)
 	return 0;
 }
 
-static int freerdp_client_set_window_size(wfContext* wfc, int width, int height)
+int freerdp_client_set_window_size(wfContext* wfc, int width, int height)
 {
```

Comment:
what's wrong this static declaration ?

Reply:
@hardening there is a `FREERDP_API` declaration for this in the header